### DISCUSSION TITLE
Nerd font not working

### DISCUSSION THREAD

**Discussion by caniszczyk** [starship/starship#2654] [Q&A] [answered]
When I do this: echo -e "\xee\x82\xa0"                      

I get a question mark + block. I have nerd fonts installed:

❯ brew search fira      
==> Casks
font-fira-code ✔                 font-fira-mono ✔                 font-fira-mono-nerd-font ✔       font-fira-sans-condensed         font-firago
font-fira-code-nerd-font         font-fira-mono-for-powerline     font-fira-sans                   font-fira-sans-extra-condensed

Any idea what's going on or what I did wrong?

**Comment by vladimyr** [comment, marked as answer]
You need to make sure that `font-fira-mono-nerd-font` is actually set as your iTerm2, Terminal.app, VSCode, terminal emulator of your choice, font.

**Comment by caniszczyk** [comment]
Ah thanks, I'm an idiot, that fixed the problem.



On Thu, Apr 29, 2021 at 3:06 AM Dario Vladović ***@***.***>
wrote:

> You need to make sure that font-fira-mono-nerd-font is actually set as
> your iTerm2, Terminal.app, VSCode, terminal emulator of your choice, font.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/starship/starship/discussions/2654#discussioncomment-673605>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAPSIPPUUD32M7TFKJYLYLTLEHPDANCNFSM43VMKLWA>
> .
>


-- 
Cheers,

Chris Aniszczyk
http://aniszczyk.org

**Reply by vladimyr** [reply]
Happens to everyone :wink: Glad it works now!
